src/daemon/rpmostreed-deployment-utils.c

```diff
@@ -133,12 +133,21 @@ variant_add_commit_details (GVariantDict *dict,
 {
   g_autoptr(GVariant) metadata = NULL;
   g_autofree gchar *version_commit = NULL;
+  g_autofree gchar *end_of_life = NULL;
   guint64 timestamp = 0;
 
   timestamp = ostree_commit_get_timestamp (commit);
   metadata = g_variant_get_child_value (commit, 0);
   if (metadata != NULL)
-    g_variant_lookup (metadata, "version", "s", &version_commit);
+    {
+      g_variant_lookup (metadata, "version", "s", &version_commit);
+      /* avoid double insertion */
+      if (prefix != NULL)
```

Comment:
do we only want base commit deployment to have this end-of-life attribute, or all type of deployments should contain end-of-life attribute if they exist